Automattic/jetpack
# WPCOM API: update option names in request format

#### Changes proposed in this Pull Request:

* I had forgotten to do it in #16568. We must update this as well so the new options are considered valid for update.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to https://developer.wordpress.com/docs/api/console/
* Try to update these 2 options for your site running this patch.

<img width="1246" alt="screenshot 2020-11-26 at 19 31 26" src="https://user-images.githubusercontent.com/426388/100383539-18e8e380-301e-11eb-99e5-df4c52b2b7c9.png">

* Visit https://wordpress.com/settings/discussion/ and notice that the settings are reflected there.

#### Proposed changelog entry for your changes:

* N/A
